dev-docs/bidders/easybid.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: bidder
+title: Easybid
+description: Prebid Easybid Bidder Adapter
+pbjs: true
+biddercode: easybid
+gvl_id: 1068
+tcfeu_supported: true
+usp_supported: true
+gpp_supported: true
+schain_supported: true
+dchain_supported: false
+floors_supported: true
+userIds: all
+tcf2_supported: true
```

Comment:
tcf2_support flag is no longer supported - please remove.

Reply:
Done, updated to tcfeu_supported